modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -164,6 +176,11 @@ include::snippets/addchannels_vendor_cli.adoc[]
 
 ifeval::[{uyuni-content} == true]
 
+== Client Tools
+
+Without client tools onboarding clients will fail.
+Client tools you can only add from the command line.
```

Comment:
I'm not sure about this particular sentence. The wording is wrong (it should be `You can add client tools only from the command line`), but IMHO it's unnecessary here? I might remove this sentence.

When we can do one thing multiple times, then I'd recommend adding a preface saying `You can do X by using Y, Z, and Alpha`. The reason for that is that a customer might see multiple ways to do something, but they might think that they need to do it multiple times.

If we only have one procedure type, I think it's fine not to preface the methods for doing it.

Just my 2c.

Reply:
This makes sense.  The next generation could even think about separating webui and CLI into two separate guides.

Reply:
separate issue created